tests/regression/run_chatbot_regression.py

```
@@ -0,0 +1,187 @@
+        data = json.load(f)
+    if not isinstance(data, list):
+        raise ValueError("cases must be a JSON array")
+    return data
+
+
+def post_chatbot(url: str, text: str, token: str | None, timeout: float):
+    body = json.dumps({"text": text}).encode("utf-8")
+    headers = {"Content-Type": "application/json"}
+    if token:
+        headers["Authorization"] = f"Bearer {token}"
+    req = request.Request(url, data=body, headers=headers, method="POST")
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)